metpy/calc/kinematics.py

```diff
@@ -748,3 +748,95 @@ def absolute_vorticity(u, v, dx, dy, lats, dim_order='yx'):
     f = coriolis_parameter(lats)
     relative_vorticity = vorticity(u, v, dx, dy, dim_order=dim_order)
     return relative_vorticity + f
+
+
+@exporter.export
+@check_units('[speed]', '[speed]', '[length]', '[length]', '[dimensionless]',
+             '[temperature]', '[pressure]')
+def potential_vorticity_baroclinic(u, v, dx, dy, lats, potential_temperature, pressure,
```

Comment:
I'm wondering if this would be more consistent with the rest of the api if the call was:
```python
potential_vorticity_baroclinic(potential_temperature, pressure, u, v, dx, dy, lats)
```

It would also also us to drop the trailing parameters when we get xarray-ified.

Reply:
No change to barotropic?

Reply:
Apparently github swallowed that comment. For barotropic, I'd move `heights` to the front.